content/en/integrations/_index.md

```diff
@@ -7,6 +7,11 @@ aliases:
     - /integrations/tcp_queue_length/
     - /integrations/snyk/
 description: Gather data from all of your systems, apps, & services
+cascade:
+    algolia:
+        rank: 40
+        category: Integrations
+        subcategory: Documentation
```

Comment:
Either flip these, or remove the subcat?

Reply:
Agree to flip them so the search results will look like "Documentation > Integrations > Page Title".